modules/buildslave/manifests/install.pp

```diff
@@ -2,6 +2,9 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 class buildslave::install {
+    if ($::operatingsystem != windows) {
```

Comment:
judging by https://dxr.mozilla.org/build-central/source/puppet/modules/packages/manifests/gcc.pp#14

should we invert this and make it `if ($::operatingsystem == CentOS)`?